### DISCUSSION TITLE
Starting Hindi Translation of Docs

### DISCUSSION THREAD

**Discussion by thetejasmagade** [vuejs/docs#1910] [Show and tell]
Hey Indian vue developers, 

So, I'm starting a "**Hindi Translation**" of Vue 3. Anyone, want's to help they can. I will share github repo within 1 day for the same here in this discussion. 
It will help a lot of developers.

Thank you

**Comment by Akshit1025** [comment]
Count me in

**Reply by thetejasmagade** [reply]
Sure, Fork and pull main branch into your computer.

After that we will split out work

**Reply by Akshit1025** [reply]
fork the repo you made or the official one

**Reply by thetejasmagade** [reply]
I made that one

**Reply by vishnuchd** [reply]
i want to modify some string you translated and want to translate new one, please guide me here
